### ISSUE TITLE
[BUG] Kirby Card Header title is rendered as a H2

### ISSUE THREAD

**Issue by chi-jyskebank-dk** [kirbydesign/designsystem#3809]
- [x ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
The title of a Kirby Card Header should not be rendered as a `<h2>`, as that will mess up the structure of the page for screen readers.

## Describe how to reproduce the bug
Insert a Kirby Card with a Kirby Card Header and see that it is rendered as a `<h2>`-tag.

1. Go to https://cookbook.kirby.design/#/home/showcase/card 
2. See that "Flagged card header" is a `<h2>`-tag

## Which Kirby version was used?
10.2.1

## What was the expected behavior?
The title should just be a plain text.

## Add any screenshots
![Image](https://github.com/user-attachments/assets/a1b52b9b-4fd1-4071-a21f-89fa8f5fd216)

## Please complete the following information:
n/a

## Are there any additional context?
n/a

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [ ] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/develop/.github/SUPPORT.md#team-kirby); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.
